tools/bazel.sh

```diff
@@ -18,5 +18,28 @@ if [ ! -d "${INSTALL_BASE}" ]; then
   done
 fi
 
-# Finally run Bazel with the new intall base
-"${BASEDIR}/bin/bazel-real" --install_base="${INSTALL_BASE}" "$@"
+export BAZEL_REAL="${BASEDIR}/bin/bazel-real"
+
+
+WORKSPACE_DIR="${PWD}"
+while [ "${WORKSPACE_DIR}" != / ]; do
+    if [ -e "${WORKSPACE_DIR}/WORKSPACE" ]; then
+      break;
+    fi
+    WORKSPACE_DIR="$(dirname "${WORKSPACE_DIR}")"
+done
+readonly WORKSPACE_DIR
```

Comment:
What does the "readonly" protect against in a ~40 line shell script that fits completely on one screen? :)

Reply:
This is just to be consistent with [production bazel script](https://github.com/bazelbuild/bazel/blob/7be6ec66422e23705a8d8345675ee4c15bad3882/scripts/packages/bazel.sh#L73)